compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -151,6 +151,26 @@ OMR::SymbolReferenceTable::findOrCreateContiguousArraySizeSymbolRef()
    return element(contiguousArraySizeSymbol);
    }
 
+#if defined(J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION)
+TR::SymbolReference *
+OMR::SymbolReferenceTable::findOrCreateContiguousArrayDataAddrFieldShadowSymRef()
+   {
+   if (!element(contiguousArrayDataAddrFieldSymbol))
+      {
+      TR::Symbol * sym = TR::Symbol::createShadow(trHeapMemory(), TR::Int32);
```

Comment:
Is there a reason why we use `Int32` here ? Isn't the data address pointer 64-bits (on 64-bit platforms which may be the only place where off-heap is done) ?

Reply:
Sorry, that should be `TR::Address`. I have updated it in my latest commit.